tools/hostbasedimagenames/main.go

```diff
@@ -0,0 +1,40 @@
+package hostbasedimagenames
+
+import (
+	"fmt"
+	"regexp"
+)
+
+var (
+	// TODO: This does not conform to https://tools.ietf.org/html/rfc3986 yet.
```

Comment:
The comment change as same to [1]

[1]: https://github.com/xiekeyang/oci-discovery/blob/master/oci_discovery/host_based_image_names/__init__.py#L18-L21

Reply:
> The comment change as same to [1][]

Addressed with e3e4231 → 3bb506e, which updated the regexp to be complete (analogous to #18).

[1]: https://github.com/xiekeyang/oci-discovery/blob/master/oci_discovery/host_based_image_names/__init__.py#L18-L21